### PR TITLE
setting up docusaurus with default configuration

### PR DESCRIPTION
**Pull request by renatoselenica**
Please check out Contributing Guidelines. By following this template you help us to review your code.
https://github.com/draft-js-plugins/draft-js-plugins/blob/master/CONTRIBUTING.md

## Checklist

- [ ] Fix any eslint errors that occur
- [ ] Update change-log for every plugin you touch
- [ ] Add/Update tests if you add/change new functionality
- [x] Add/Update docs if you add/change functionality
- [x] Enable "Allow edits from maintainers" for this PR

## Use-case/Problem

This is a PR to setup the new Docs using Docusaurus. The setup is using the default configuration so we can now work on firstly adding up the information from the existing docs.
The way Docusaurus works is by having all the docs in markdown files in a folder which in this case is called docsV2, and having a website folder for the actual docs website where the markdown gets transformed into static files.
The Docusaurus documentation on how to set everything up and work on different elements is very clear so it is easy for all of us who want to, to contribute to the new docs.
I haven't modified the README yet so I am writing here how to get started:

`cd website`
To start the development server:
`yarn run start`
To build the site:
`yarn run build`

P.S. I am a bit busy on the weekend so I will start adding the old documentation next week.
P.S.S. The Linting rules complain a lot on the code on the generated website, I can fix them, but I just don't know how that would effect Docusaurus. Should I just disable eslint on that folder for now (that doesn't seem like a great solution either) Awaiting an opinion on this.

### CODE REVIEW

**Comment by juliankrispel** [comment]
@renatoselenica I'd rather not disable linting for now but I also wouldn't worry about it too much. We can work on this pr together and someone else can fix the linting issues, it shouldn't affect docusaurus at all since they're just code style ...

File: docsV2/doc1.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Phasellus
+
+Phasellus pulvinar ex id commodo imperdiet. Praesent odio nibh, sollicitudin sit amet faucibus id, placerat at metus. Donec vitae eros vitae tortor hendrerit finibus. Interdum et malesuada fames ac ante ipsum primis in faucibus. Quisque vitae purus dolor. Duis suscipit ac nulla et finibus. Phasellus ac sem sed dui dictum gravida. Phasellus eleifend vestibulum facilisis. Integer pharetra nec enim vitae mattis. Duis auctor, lectus quis condimentum bibendum, nunc dolor aliquam massa, id bibendum orci velit quis magna. Ut volutpat nulla nunc, sed interdum magna condimentum non. Sed urna metus, scelerisque vitae consectetur a, feugiat quis magna. Donec dignissim ornare nisl, eget tempor risus malesuada quis.
```
[the remark this answers is not in the document]
Comment (renatoselenica) [reply docsV2/doc1.md]: Yeah it was all generated. As a sample. Will be removing those soon, the moment I start putting on our documentation

**Comment by renatoselenica** [comment]
Awesome, then will be working on adding documentation first, then fix the linting.

**Comment by juliankrispel** [comment]
Exactly!

**Comment by juliankrispel** [comment]
@renatoselenica since you're not working on this anymore I'm closing this 😢

**Comment by renatoselenica** [comment]
I am really sorry. My life and work has been a ruckus recently.

**Comment by juliankrispel** [comment]
No worries bro totally get it ❤️

**Pull request closed without merging**